### ISSUE TITLE
Highlight markers between every word after upgrade to 5.x

### ISSUE THREAD

**Issue by jacobgkau** [RocketChat/Rocket.Chat#27032]
### Description:

In the web (+ desktop app) UI, every word has a `<mark class="highlight-text"></mark>` line between it, resulting in a mess:

![Screenshot_20221008_143906](https://user-images.githubusercontent.com/7199422/194727030-dc9d834a-33e4-4d7a-b455-228c4753b13a.png)

I don't see anything wrong when accessing via the mobile app, so I don't think the messages are actually malformed in the database or anything. This affects messages from both before and after the upgrade was performed.

### Steps to reproduce:

1. Upgrade a 4.x instance (that was previously upgraded from 3.x) to 5.x

### Expected behavior:

Messages display properly.

### Actual behavior:

See screenshot in the above description.

### Server Setup Information:

- Version of Rocket.Chat Server: Currently `5.0.6` from the `5.x/stable` snap channel.
- Operating System: Ubuntu 20.04 LTS
- Deployment Method: snap
- Number of Running Instances: 1
- DB Replicaset Oplog: ???
- NodeJS Version: See snap version
- MongoDB Version: See snap version

### Client Setup Information

- Desktop App or Browser Version: Both (desktop app version 3.8.11.)
- Operating System: Arch Linux

**Comment by dudanogueira** [comment]
Hi! Thanks for pointing it out.

I was not able to reproduce this on docker =\

I tried, however, to set a space as a highlighted text, but also I was not able. Setting this option to something like
`this, ,test`

will render it as
```
this,
test
```
after saving

can you check what you have configured as your highlight config? also, can you try changing it and see if the issue go away? Maybe some migration allowed a space to be considered

I will try to follow the snap path you mentioned later this week.

**Comment by jacobgkau** [comment]
I found the setting you were talking about (user-level setting in My Account -> Preferences):
![image](https://user-images.githubusercontent.com/7199422/195259941-a9a23d58-525b-42dd-98cc-b851556e458b.png)
Above is what I had configured before (there was nothing in the box, as I'd never configured anything.) I tried setting the option to "test" and saved, and the issue went away. I then removed everything from the box and saved again, and the issue did not come back.

I will need to check if my other users are also affected by the issue. If they are, I'll need to figure out how to reset this setting for all of them at once via the CLI so they don't all have to toggle the setting themselves.

**Comment by jacobgkau** [comment]
I've confirmed with one of my server's other users that they are not experiencing this issue. I'm not sure why it would have happened for only my account, but my account would have the most history out of everyone on the server, as I've used it to make YouTube videos about RocketChat in the past (and I might have demonstrated various settings while doing that.)

I'll close this issue for the time being. It can be re-opened if someone else experiences it after an upgrade.
